my.css

```diff
@@ -163,3 +164,159 @@ body {
   display: flex;
   justify-content: center;
 }
+@media screen and (max-width: 1130px) {
+	.collection {
+	max-height: 700px;
+	position: relative;
+	padding-top: 100px;
+  display: flex;
+  justify-content: center;
+  max-width: 1150px;
+  height: 750px;
+  z-index: 1;
+  margin: 0 auto;
+    }
+    .wraper::before {
+	content: '';
+	position: absolute;
+	left: 0;
+	top: 0;
+	width: 400px;
+	height: 800px;
+	background-color: black;
+	z-index: -1;
+    }
+    .burgermenu {
+  position: relative; 
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5608084)